pom.xml

```diff
@@ -3179,7 +3179,336 @@
+                                                    <property>
+                                                        <name>includes</name>
+                                                    </property>
+                                                    <property>
+                                                        <name>outputEncoding</name>
+                                                    </property>
+                                                    <property>
+                                                        <name>sourceEncoding</name>
+                                                    </property>
+                                                    <property>
+                                                        <name>title</name>
+                                                    </property>
+                                                </properties>
+                                                <ignoredProperties>
+                                                    <ignore>project</ignore>
+                                                    <ignore>reactorProjects</ignore>
+                                                    <!--
+                                                     we should not be ignoring this since formats changes the content of the output
+                                                      but it contains a list of enum see https://github.com/jacoco/jacoco/blob/562a7299399b8321dfcf8eeb21ea4bfc27556fdc/jacoco-maven-plugin/src/org/jacoco/maven/AbstractReportMojo.java
+                                                     and that's not storable in the cache key
+                                                     no sure how this should be handled ?
+                                                     -->
+                                                    <ignore>formats</ignore>
+                                                </ignoredProperties>
+                                            </inputs>
+                                            <outputs>
+                                                <cacheableBecause>compilation is CPU-bound</cacheableBecause>
+                                                <directories>
+                                                    <directory>
+                                                        <name>outputDirectory</name>
+                                                    </directory>
+                                                </directories>
+                                            </outputs>
+                                        </execution>
+                                    </executions>
+                                </plugin>
                             </plugins>
+                            <normalization>
+                                <runtimeClassPath>
+                                    <ignoredFiles>
+                                        <ignoredFile>git.properties</ignoredFile>
```

Comment:
I haven't been able to retrieve the contents to understand why they were different but [build scan comparison](https://develocity.apache.org/c/iy5unsfnfjnzo/qa2fkmefkbzkk/goal-inputs?expanded=WyJwMmtjeXhvMjQycWF5LXRlc3RjbGFzc3BhdGhlbGVtZW50cyIsInAya2N5eG8yNDJxYXktdGVzdENsYXNzcGF0aEVsZW1lbnRzLTAtMCIsIm53dHByajJjdXB5Mm8tdGVzdGNsYXNzcGF0aGVsZW1lbnRzIiwibnd0cHJqMmN1cHkyby10ZXN0Q2xhc3NwYXRoRWxlbWVudHMtMC0xIiwibnd0cHJqMmN1cHkyby10ZXN0Q2xhc3NwYXRoRWxlbWVudHMtMC0wIl0) showed that this file was preventing caching of the test outputs